libmamba/src/api/configuration.cpp

```diff
@@ -1776,11 +1776,12 @@ namespace mamba
                                          ctx.prefix_params.root_prefix / "condarc.d",
                                          ctx.prefix_params.root_prefix / ".mambarc" };
 
-        std::vector<fs::u8path> home = { env::home_directory() / ".conda/.condarc",
-                                         env::home_directory() / ".conda/condarc",
-                                         env::home_directory() / ".conda/condarc.d",
-                                         env::home_directory() / ".condarc",
-                                         env::home_directory() / ".mambarc" };
+        std::vector<fs::u8path> home = {
+            env::home_directory() / ".conda/.condarc",   env::home_directory() / ".conda/condarc",
+            env::home_directory() / ".conda/condarc.d",  env::home_directory() / ".condarc",
+            env::user_config_dir() / "../conda/condarc", env::home_directory() / ".mambarc",
```

Comment:
@jonashaag Because `user_config_dir` is (default) `$XDG_CONFIG_HOME/mamba` but the last review wanted `$XDG_CONFIG_HOME/conda` and this seems like the best way to make it conda/mamba compatible.  Otherwise I would have to set `user_config_dir` to either be just `$XDG_CONFIG_HOME` and always supply `mamba` after calling it, or I would have to give it a `mamba` argument, all so I can supply `conda` in a few default cases.  it seems like `../conda` is an easier solution

Reply:
Sorry, should have been clear, I wanted a _code comment_ with more or less exactly this content :)